src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/CustomImportOrderCheckTest.java

```diff
@@ -159,26 +159,7 @@ public void testSamePackageAndStatic() throws Exception {
     }
 
     @Test
-    public void testOnlySamePackage() throws Exception {
```

Comment:
This UT was removed as it is covered by new UTs (testSamePackageDepth2, testSamePackageDepth3, ...)